packages/truffle-workflow-compile/test/index.js

```diff
@@ -25,30 +36,28 @@ describe("Contracts.compileSources", () => {
 });
 
 describe("Contracts.compile", () => {
-  it("when config.all is true, all contracts in contracts_directory are recompiled", () => {
+  it("when config.all is true, all contracts in contracts_directory are recompiled", async () => {
     let contractName;
 
     // initial compile
-    Contracts.compile(config, (err, { contracts }) => {
+    await Contracts.compile(config, (err, { contracts }) => {
       if (err) assert.fail(err);
-      else {
-        contractName = Object.keys(contracts)[0];
-        assert(
-          existsSync(`${config.contracts_build_directory}/${contractName}.json`)
-        );
-      }
+      contractName = Object.keys(contracts)[0];
+      assert(
+        existsSync(`${config.contracts_build_directory}/${contractName}.json`)
+      );
     });
 
     // compile again
     config.all = true;
-    Contracts.compile(config, (err, output) => {
+    await Contracts.compile(config, (err, { outputs }) => {
```

Comment:
I think the singular `output` is used elsewhere. Any reason it is plural here?

Reply:
Same here.

Reply:
@fainashalts it looks like `output` is only used in the README in the context of logging the entire `output` object.

it's plural because of the object structure Tyler is showing above. the fancy ES6 destructuring returns the `outputs` key directly rather than having to do something like `output.outputs.solc[0]`.

Reply:
Ah I see now. Thanks for clarifying!